packages/gatsby-plugin-postcss-sass/package.json

```diff
@@ -28,8 +28,5 @@
   "main": "index.js",
   "readme": "README.md",
   "scripts": {
-    "build": "babel src --out-dir . --ignore __tests__",
-    "watch": "babel -w src --out-dir . --ignore __tests__",
-    "prepublish": "cross-env NODE_ENV=production npm run build"
```

Comment:
this whole package should be deleted, the files here are leftover from a bad merge with master